docs/guides/query-kyc-state-on-c-chain.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_position: 45
+title: Query KYC State on C-Chain
+description: This document describes how to use Remix IDE to query KYC State of an address on C-Chain
+---
+
+# How to Query KYC State on C-Chain
+
+This document provides a guide on how to use Remix IDE to query the KYC state of
+an address on C-Chain. Follow the steps below to accomplish this.
+
+## Requirements
```

Comment:
Might be useful to add to the requirements:
"
- Remix IDE
- CaminoETHVM ABI file  
"

Reply:
These are not prerequisites, the document handles these. Maybe I should change the title to Prerequisites instead of Requirements?